### PR TITLE
Update AboutUs page--wangtao

### LINKED ISSUE #1: Update AboutUs page (week 7) - Wang Tao

### PR DESCRIPTION
**Pull request by wangtao0717**
Fix #1

### CODE REVIEW

**Review by wangtao0717 — COMMENTED** [review]
fix #3

File: docs/UserGuide.md [hunk 1]
```diff
@@ -73,19 +73,15 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a apparent: `add`
```
Comment (whatthelump) [inline docs/UserGuide.md]: Hi @wangtao0717 very minor typo here, will approve when this is fixed!

**Review by VRSoorya — APPROVED** [review]
LGTM!

**Review by VRSoorya — APPROVED** [review]
your image file awaits!

**Pull request merged**